vector/src/test/java/im/vector/app/test/fakes/FakeSession.kt

```diff
@@ -34,10 +35,13 @@ class FakeSession(
         mockkStatic("im.vector.app.core.extensions.SessionKt")
     }
 
+    override val myUserId: String = "a-user-id"
```

Comment:
I am wondering if we should have a valid userId here like `@fake:server.fake`. WDYT?

Reply:
sounds great to me! makes me wonder if a dedicated type for `UserId` or `MatrixId` could be helpful (might raise a ticket for a [value class](https://kotlinlang.org/docs/inline-classes.html) PoC)

Reply:
ah it already exists https://github.com/vector-im/element-android/issues/3633

https://github.com/vector-im/element-android/pull/5323/commits/deec923026c8a6d158e9396f961b8c1a8284e69f for the value update